### DISCUSSION TITLE
Ukrainian language support

### DISCUSSION THREAD

**Discussion by blevkovych** [coqui-ai/TTS#4050] [Ideas]
I see that you support Polish, Hungarian, Czech, Russian. Could I please ask to add aa Ukrainian language support.
Also your Polish throws errors, https://huggingface.co/spaces/coqui/xtts

**Comment by patriotyk** [comment]
Forget it, they don't work on it anymore, so will not add new languages. But it is easy to train it yourself. But If you need Ukrainian you can also use styletts2 -  https://huggingface.co/spaces/patriotyk/styletts2-ukrainian it generates even better.

**Reply by karu2003** [reply]
Is this service online only?

**Reply by patriotyk** [reply]
No
